reference/twig_reference.rst

```diff
@@ -371,18 +372,22 @@ absolute_url
 ``path``
     **type**: ``string``
 
-Returns the absolute URL for the given absolute path. This is useful to convert
-an existing path:
+Returns the absolute URL from the passed relative path. Assume the following
+code was used to render ``http://example.com/sub/list.html``:
 
 .. code-block:: jinja
 
-    {{ absolute_url(asset(path)) }}
+    {{ absolute_url('/human.txt') }}
```

Comment:
Doing it twice seems a bit duplicated to me. I've not included it in the PR, but feel free to make this change during merging if you feel it makes things better.